### PR TITLE
Move to apps org and new repo.  Updated all references.

### PR DESCRIPTION
**Pull request by deanesmith**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ pipeline {
           HELM_RELEASE = "$PREVIEW_NAMESPACE".toLowerCase()
         }
         steps {
-          dir ('/home/jenkins/go/src/github.com/jenkins-x/ext-jacoco') {
+          dir ('/home/jenkins/go/src/github.com/jenkins-x-apps/jx-app-jacoco') {
```
Comment (jglick) [inline Jenkinsfile:21]: BTW all this is nuts. If you use Go modules, you do not need to check out to a specific directory, so all the `dir` steps could either be omitted or take relative paths. (Even in the latter case, something like `make -C charts/jx-app-jacoco` is more straightforward.)

Discussion in https://github.com/jenkins-x/draft-packs/issues/83.

File: charts/jx-app-jacoco/values.yaml [hunk 2]
```diff
@@ -3,11 +3,11 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: jenkinsxio/ext-jacoco
+  repository: draft
```
Comment (jglick) [inline charts/jx-app-jacoco/values.yaml]: ?

File: charts/preview/Makefile [hunk 3]
```diff
@@ -8,7 +8,7 @@ ifeq ($(OS),Darwin)
 else ifeq ($(OS),Linux)
 	sed -i -e "s/version:.*/version: $(PREVIEW_VERSION)/" Chart.yaml
 	sed -i -e "s/version:.*/version: $(PREVIEW_VERSION)/" ../*/Chart.yaml
-	sed -i -e "s|repository: .*|repository: $(DOCKER_REGISTRY)\/jenkins-x\/ext-jacoco|" values.yaml
+	sed -i -e "s|repository: .*|repository: $(DOCKER_REGISTRY)\/jenkins-x-apps\/jx-app-jacoco|" values.yaml
```
Comment (jglick) [inline charts/preview/Makefile:11]: BTW https://github.com/jenkins-x-buildpacks/jenkins-x-kubernetes/pull/12

**Review by jglick — APPROVED** [review]
Wow there is a lot of boilerplate to update here. I wonder if buildpacks can be improved to deduplicate somewhat, and even infer `GITHUB_ORG` and `APP_NAME` from the checkout location.

File: Jenkinsfile (hunk 1 above)
Comment (rawlingsj) [reply Jenkinsfile:21]: yeah at the tome of creating the Jenkinsfiles we didn't use go modules, would be good to switch.

Comment (deanesmith) [reply Jenkinsfile:21]: @jglick I will create a separate PR for that as part of our improvements.

File: charts/jx-app-jacoco/values.yaml (hunk 2 above)
Comment (rawlingsj) [reply charts/jx-app-jacoco/values.yaml]: yeah looks like you need `jenkinsxio/jx-app-jacoco`

Comment (deanesmith) [reply charts/jx-app-jacoco/values.yaml]: Will do.

**Comment by jglick** [comment]
BTW using Go 1.11.5 I cannot build without

```diff
diff --git a/go.sum b/go.sum
index 5defe2b..052d7e5 100644
--- a/go.sum
+++ b/go.sum
@@ -230,7 +230,7 @@ k8s.io/apiextensions-apiserver v0.0.0-20180621085152-bbc52469f98b h1:q6seqITE2eU
 k8s.io/apiextensions-apiserver v0.0.0-20180621085152-bbc52469f98b/go.mod h1:IxkesAMoaCRoLrPJdZNZUQp9NfZnzqaVzLhb2VEQzXE=
 k8s.io/apimachinery v0.0.0-20180621070125-103fd098999d h1:MZjlsu9igBoVPZkXpIGoxI6EonqNsXXZU7hhvfQLkd4=
 k8s.io/apimachinery v0.0.0-20180621070125-103fd098999d/go.mod h1:ccL7Eh7zubPUSh9A3USN90/OzHNSVN6zxzde07TDCL0=
-k8s.io/client-go v8.0.0+incompatible h1:7Zl+OVXn0bobcsi4NEZGdoQDTE9ij1zPMfM21+yqQsM=
+k8s.io/client-go v8.0.0+incompatible h1:tTI4hRmb1DRMl4fG6Vclfdi6nTM82oIrTT7HfitmxC4=
 k8s.io/client-go v8.0.0+incompatible/go.mod h1:7vJpHMYJwNQCWgzmNV+VYUl1zCObLyodBc8nIyt8L5s=
 k8s.io/code-generator v0.0.0-20180621065459-6702109cc68e/go.mod h1:MYiN+ZJZ9HkETbgVZdWw2AsuAi9PZ4V80cwfuf2axe8=
 k8s.io/gengo v0.0.0-20180718083919-906d99f89cd6/go.mod h1:ezvh/TsK7cY6rbqRK0oQQ8IAqLxYwwyPxAX1Pzy0ii0=
```

Should we just stop checking `go.sum`s into source control? They seem to be a constant hassle.

**Comment by jglick** [comment]
Sounds like https://github.com/golang/go/issues/27925 and the constellation of related issues. I removed the offending line from `go.sum`, did a `go clean -modcache`, and did a new build, but now I have a far larger diff, including apparently unrelated things in `go.mod` like

```diff
-	github.com/jenkins-x/jx v1.3.445
+	github.com/jenkins-x/jx v1.3.807
```

WTF?

**Pull request merged**